data_structures/linked_list/singly_link_list_deletion.c

```diff
@@ -73,11 +88,11 @@ void viewlist()  // function to display values
 
 int main()
 {
-    int n;
+    int n=0,pos=0;
     while (1)
     {
         printf("\n1.add value at first location");
```

Comment:
Would you like to ask the user for a position for the addition process?

Reply:
I only changed the deletion part in this pull request.  So I left the addition part(at first position)as it is in the previous code. But I can try enhancing the addition part also to any position if assigned

Reply:
Feel free to work on that if you want. 🙂

Reply:
Sure! I will try that else will include it in a separate pull request